### DISCUSSION TITLE
Version 2.2

### DISCUSSION THREAD

**Discussion by hossein1376** [hossein1376/English-Persian-Kindle-Custom-Dictionary#2] [Announcements]
**تغییرات:**

اضافه شدن کلمات جدید

بهبود inflection های موجود

بهبود املا، ترتیب و عنوان کلمات ثبت شده

اضافه کردن inflection برای بیش از 1500 فعل

**تاریخ انتشار:**

 01/13/2022

<hr /><em>This discussion was created from the release <a href='https://github.com/hossein1376/English-Persian-Kindle-Custom-Dictionary/releases/tag/v2.2'>Version 2.2</a>.</em>

**Comment by Bana6** [comment]
damet garm . kheili donbalesh boodam. baraye kobo ham kar mikone?
